intervention_compassion/views/compassion_intervention_view.xml

```diff
@@ -90,6 +90,8 @@
                                     <field name="estimated_costs" widget="monetary" options="{'currency_field': 'currency_usd'}"/>
                                     <field name="estimated_impacted_beneficiaries"/>
                                     <field name="estimated_local_contribution" widget="monetary" options="{'currency_field': 'currency_usd'}"/>
+                                    <field name="local_currency"/>
```

Comment:
You can make it `invisible="1"` since we already see the currency in the cost field.